phoenix-core/src/main/java/org/apache/phoenix/schema/MetaDataClient.java

```diff
@@ -3707,6 +3738,13 @@ public MutationState addColumn(PTable table, List<ColumnDef> origColumnDefs,
                             }
                             colFamiliesForPColumnsToBeAdded.add(column.getFamilyName() == null ? null : column.getFamilyName().getString());
                             addColumnMutation(schemaName, tableName, column, colUpsert, null, pkName, keySeq, table.getBucketNum() != null);
+                            // add new columns for given indexes one by one
+                            if (cascade) {
+                                for (PTable index: indexesPTable) {
+                                    logger.info("Adding column to "+index.getTableName().toString());
```

Comment:
Please add the column name here to make this log-line really useful.

Reply:
This was intended more for test debugging. Detailed logging to follow.